### DISCUSSION TITLE
What's the difference between "go-orb, previously go-micro" (let's say they are one single project) compared with the similarly named project: "https://github.com/micro/micro"?

### DISCUSSION THREAD

**Discussion by dievilz** [go-orb/go-orb#14] [General]
Pretty much the title.
I was looking for a project I stumbled upon a lil while ago (https://github.com/go-kit/kit) because I liked their architecture, but I couldn't remember the name. I knew it wasn't Micro (https://github.com/micro/micro), but when I found go-micro and go-orb, I thought you were the ones I was looking for, sadly you weren't 😅. But the question in the title arose thinking about what if I were to choose one project between both.

Thanks in advance.

**Comment by jochumdev** [comment]
Hi @dievilz 

go-orb is the fresh go-micro, nothing else.

See the README of orb about it's features.

Kind regards,
René

**Comment by jochumdev** [comment]
micro/micro is a platform for hosted micro services. Where go-micro and go-orb are frameworks. Go-Orb will be the successor of go-micro, once it's done.
